go.mod

```diff
@@ -5,27 +5,41 @@ go 1.15
 require (
 	github.com/blang/semver/v4 v4.0.0
 	github.com/coreos/prometheus-operator v0.41.1
+	github.com/davecgh/go-spew v1.1.1
+	github.com/fsnotify/fsnotify v1.4.9
 	github.com/ghodss/yaml v1.0.0
+	github.com/go-kit/kit v0.10.0
 	github.com/go-logr/logr v0.3.0
 	github.com/onsi/ginkgo v1.14.2
 	github.com/onsi/gomega v1.10.4
-	github.com/openshift/api v0.0.0-20200930075302-db52bc4ef99f // release-4.6
+	github.com/openshift/api v0.0.0 // release-4.6
+	github.com/openshift/client-go v0.0.0
 	github.com/openshift/custom-resource-status v0.0.0-20200602122900-c002fd1547ca
 	github.com/operator-framework/api v0.5.3
 	github.com/operator-framework/operator-lib v0.4.0
 	github.com/spf13/cobra v1.1.1
+	github.com/spf13/pflag v1.0.5
 	gomodules.xyz/jsonpatch/v2 v2.1.0
 	gopkg.in/yaml.v2 v2.3.0
-	k8s.io/api v0.20.2
-	k8s.io/apiextensions-apiserver v0.20.1
-	k8s.io/apimachinery v0.20.2
+	k8s.io/api v0.20.4
+	k8s.io/apiextensions-apiserver v0.20.2
```

Comment:
Can this be: `k8s.io/apiextensions-apiserver v0.20.4`? To be the same version as other k8s.io api clackages.

Reply:
Corrected